src/_base/application/overlay/auth.json.twig

```diff
@@ -0,0 +1,15 @@
+{
+    "http-basic": {
+      {% for repo in @('composer.auth.basic') %}
+        "{{ repo.path }}": {
+            "username": "{{ repo.username }}",
+            "password": "{{ repo.password }}"
+        }{% if not loop.last %},{% endif %}
+      {% endfor %}
+    },
+    "github-oauth": {
+      {% if @('composer.auth.github') %}
+        "github.com": "{{ @('composer.auth.github') }}"
+      {% endif %}
+    }
+}
```

Comment:
we don't use composer in node, so this auth.json isn't needed, along with the composer.* variables related to it if in other files

Reply:
fixed: removed references and functions related to Composer. thank you